heatmap/node-taint.html

```diff
@@ -1,35 +1,61 @@
 <html>
-<head>
-<style>
-h1 {text-align: center;color:#339933;}
-h3 {text-align: center;color:#339933;}
-p {text-align: center;}
-div {text-align: center;}
-.aligncenter {
-    text-align: center;
-}
-</style>
-</head>
-<body>
-
-<h1><u>Node Taint Chaos</u></h1>
-<br>
-<p>Node Taint exhaust memory resources on the kubernetes node. Here are the different coverage metrics for the experiment.
-<br><br>
-
-<h3><i> Node Taint experiment based coverage<i></h3>
-<div>
-<img src="../images/node-taint-heatmap.png" class="aligncenter" alt="Node Taint Heatmap coverage" width="1100" height="700">
-</div>
-
-<h3><i> Node Taint cluster runtime based coverage<i></h3>
-<div>
-<img src="../images/node-taint-runtime-heatmap.png" class="aligncenter" alt="Node Taint Cluster Runtime Heatmap coverage" width="1000" height="500">
-</div>
-<h3><i> Node Taint platform based coverage<i></h3>
-<div>
-<img src="../images/node-taint-platform-heatmap.png" class="aligncenter" alt="Node Taint Platform Heatmap coverage" width="1100" height="700">
-</div>
-</p>
-</body>
-</html>
+  <head>
+    <link rel="stylesheet" href="mystyle.css">
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script type = "text/javascript" src="function.js">
+      google.charts.load('current', {packages: ['corechart', 'bar']});
+      google.charts.load('current', {'packages':['table']});
+    </script>
+  </head>
+  <body>
+    <h1><u>Node Taint Chaos</u></h1>
+    <br>
+    <p> Node Taint platform tested</p>
+    <br>
+    <div style="width: 1000px; text-align: center; margin: 0 auto;">
+      <img src="../images/platforms/GKE.png" alt="HTML5 Icon" style="width:60px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/EKS.png" alt="HTML5 Icon" style="width:140px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/AKS.svg" alt="HTML5 Icon" style="width:80px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/kubeadm.png" alt="HTML5 Icon" style="width:100px;height:60px; margin-right: 5px;">
+    </div>
+    <br><br>
+    <h1><u>Node Taint Experiment Details</u></h1>
+    <p>
+      Node Taint exhaust memory resources on the kubernetes node. 
```

Comment:
Is it correct? @uditgaurav

Reply:
needs to be updated! You can refer from [experiment metadata](https://docs.litmuschaos.io/docs/pod-delete/#experiment-metadata) @oumkale

Reply:
Thank you @uditgaurav and @Jonsy13 for pointing out. have updated.